components/question/Outro/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+
+import Image from '@/components/Image';
+import circleCheckGreenAnimation from '@/public/circle-check-green-animation.gif';
+
+export const surveyOutroTestIds = {
+  base: 'survey-outro',
+  animation: 'survey-outro__animation',
+  text: 'survey-outro__text',
+};
+
+const SurveyOutro = () => {
```

Comment:
Not sure whether we should rename this component or move it to the `survey` folder as it might be misleading as it is under the `question` folder 💭

Reply:
sure, moved it inside the `survey` folder in fff5bbb3cdc260dcce473d61ad137372948dc0fa, Thanks!